server/service.go

```diff
@@ -162,6 +163,14 @@ func CAKeyPassword(pw []byte) ServiceOption {
 	}
 }
 
+// optional argument to set the validity of signed client certs in days
```

Comment:
The comment should start with `// ClientValidity sets the ....` The name of the function should come first as per the linter.